### PR TITLE
Pomodoro mode implementation for toggl-button

### PR DESCRIPTION
**Pull request by Stvad**
I've implemented support for pomodoro-style (https://en.wikipedia.org/wiki/Pomodoro_Technique) regime for toggl-button.
The idea of Pomodoro Technique is to work in limited focused periods of time. So if you enable this mode in the options - you start the tracking work for preset period of time. After that - the tracking would be automatically stopped and you will see the notification (and hear sound signal if you choose so) that will tell you that it's time to take a short break.

### CODE REVIEW

File: src/html/settings.html [hunk 1]
```diff
@@ -18,7 +18,7 @@
         <li class="nag-nanny">
           <input type="checkbox" id="nag-nanny">
           <label for="nag-nanny">Remind me to track</label>
-          <div id="nag-nanny-details">
+          <div id="subsettings-details">
```
Comment (wmertens) [inline src/html/settings.html]: duplicate id with below?

**Comment by wmertens** [comment]
Nice! :+1: (except that duplicate id?)

**Comment by Stvad** [comment]
@wmertens Thanks for pointing that out. There should really have been class-es. I've updated the code.

**Comment by wmertens** [comment]
:+1:

**Comment by IndrekV** [comment]
Thank you very much for your hard work and your contribution to Toggl Button

**Comment by cassus** [comment]
:+1:

**Pull request merged**